app/views/layouts/searches_redesign.html.haml

```diff
@@ -15,6 +15,6 @@
     = javascript_pack_tag 'application'
     = stylesheet_pack_tag 'application'
   %body
-    #main-content= yield
+    #serp-main-content= yield
```

Comment:
SRCH-4879: Updated the reference as `#main-content` is being used at Skip to main button https://github.com/GSA/search-gov/blob/main/app/javascript/components/Header/index.tsx#L57